guardian/models.py

```diff
@@ -84,18 +84,19 @@ class GroupObjectPermissionBase(BaseObjectPermission):
     **Manager**: :manager:`GroupObjectPermissionManager`
     """
     group = models.ForeignKey(Group, on_delete=models.CASCADE)
+    origin = models.ForeignKey(Origin, on_delete=models.CASCADE, null=True)
```

Comment:
We don't use group, but tests fail now without origin and it is best to have the origin in both places anyways for consistency.